src/auto/checksum.rs

```diff
@@ -3,43 +3,43 @@
 // DO NOT EDIT
 
 use ChecksumType;
-use ffi;
+use sys;
```

Comment:
If we're changing this anyway, shouldn't we directly name this `glib_sys` instead? That way there's no renaming necessary at all even for the "local" FFI bindings and it would be more consistent.

Reply:
Currently I think that all concrete classes better be almost same as generated code (use `sys` for current FFI-crate).
In glib has 2 main parts: currently glib-based object has sys+gobject_sys,
only gobject-based classes has glib_sys+gobject_sys to minimize confusion.
If we want use `xx_sys` instead `sys` in generated code too, it maybe need loot changes in gir.

Reply:
I'd prefer to go without `sys` and rename it all to `xxx_sys`.

As with Rust 2018 the `extern crate` are not required anymore, it would be really nice if we wouldn't have to do renames. It simplifies things a lot :)

Reply:
Ok I try change this too.

Reply:
Great, thanks :)